docs/syncer.md

```diff
@@ -23,7 +23,7 @@ Current workspace is "root:my-org" (type "root:organization").
 1. Enable the syncer for a new cluster
 
 ```sh
-$ kubectl kcp workload sync <mycluster> --syncer-image <image name> -o syncer.yaml
+$ kubectl kcp workload sync <mycluster> --syncer-image <image name>  --dns-image <image name> -o syncer.yaml
```

Comment:
why is this a different image? Can we merge them?

Reply:
It is a different image because CoreDNS needs root access (see https://github.com/coredns/coredns/issues/1767#issuecomment-507213651) whereas the syncer does not. 

Instead of `--syncer-image` and `--dns-image` we could replace them by `--container-repository` and `--container-tag` and use fixed container image name.

Reply:
why does root access have anything to do with the image?

Reply:
To run non-root containers, the image needs to be built with a non-root user and the k8s deployment needs to be configured to use the uid created in the image (see `securityContext.runAsUser`). Adding a non-root user in the image is not strictly required and could work if the syncer does not access the user directory.

Reply:
There might be another solution based on container port mapping.

Reply:
container port mapping works, problem solved. I'll use one image.